README.md

```diff
@@ -111,6 +111,16 @@ function sketch(p5) {
 }
 
 function App() {
+  const [rotation, setRotation] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(rotation => setRotation(rotation + 1), 100);
```

Comment:
@yevdyko maybe this isn't a good example but it fixes the bug, we could replace this with something else, WDYT?

Reply:
It's still broken:
- `useState` and `useEffect` must be imported from `React`. 
- `rotation` is `NaN` in props.

Reply:
Updated the example and tested locally 👍🏻